nfpm.go

```diff
@@ -322,3 +322,15 @@ func (info *Info) GetChangeLog() (log *chglog.PackageChangeLog, err error) {
 		Entries: entries,
 	}, nil
 }
+
+type ErrSigningFailure struct {
```

Comment:
maybe just `ErrSign`, as err already implies a failure 🤔

Reply:
I actually thought quite a bit about the name. It seems like most packages name the error (the part after `Err`) to specify what went wrong or rather what type of error it was. It felt to me  `Sign` or `Signature` describes the result of a successful action instead of something that went wrong. And because I couldn't be more specific I settled on a `SignatureFailure`.

But actually I'm overthinking it majorly 😆
It should be your call though, so what do we do?

Reply:
hmm, dunno either... you argument makes sense though... lets leave as is.